Formula/git-tracker.rb

```diff
@@ -22,6 +22,9 @@ class GitTracker < Formula
 
   def install
     system "rake", "standalone:install", "prefix=#{prefix}"
+
+    # Replace `ruby` cellar path in shebang
+    inreplace bin/"git-tracker", Formula["ruby"].prefix.realpath, Formula["ruby"].opt_prefix unless OS.mac?
```

Comment:
Existing Linux bottle:
```
#!@@HOMEBREW_CELLAR@@/ruby/3.1.2/bin/ruby
```

New Linux bottle:
```
#!@@HOMEBREW_PREFIX@@/opt/ruby/bin/ruby
```

Existing/New Big Sur bottle and existing (unwanted) `all` bottle:
```
#!/usr/bin/ruby
```

---

Existing Linux bottle and `all` bottle are probably both broken on Linux.